app/config/parameters.yml.dist

```diff
@@ -1,12 +1,12 @@
 parameters:
-    env(DATABASE_HOST):         db
+    env(DATABASE_HOST):         enmarche_db
```

Comment:
Now, there is no need to prefix container with `enmarche_` you can revert this in all files.